websites/H/Home Assistant/presence.ts

```diff
@@ -0,0 +1,130 @@
+const presence = new Presence({
+    clientId: "812627990382772224"
+}), browsingStamp = Math.floor(Date.now() / 1000);
+  
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "home_assistant_logo", //banner
+    startTimestamp: browsingStamp
+  }; 
```

Comment:
just set the details in here...;.

Reply:
and it means to delete this function after doing it thanks.